packages/clerk-js/src/ui/polishedAppearance.ts

```diff
@@ -131,50 +131,19 @@ export const polishedAppearance: Appearance = {
         },
       },
       input: {
-        ...inputStyles(theme),
-        '&[type="radio"]': {
-          boxShadow: 'none',
-          '&:hover': {
-            boxShadow: 'none',
-          },
+        '&[data-variant="default"]': {
```

Comment:
Should we do a `&:not([data-variant="unstyled"])` to make this more future proof as well as a better example for usage in other themes? Does it make more sense to you @octoper ?

Reply:
Actually, should we just rename `unstyled` to something that makes more sense? Can we find common styles by both elements (search & tag input) and group them in a variant? Right now, we pass inline styles to both and the variant has no use, so targeting `unstyled` doesn't really make sense for any customer.

Reply:
I know this isn't part of the PR so we can do it separately if you don't want to include something here.